callback_endpoint.go

```diff
@@ -19,24 +20,52 @@ type callbackEndpoint struct {
 	errorMsg         string
 	errorDescription string
 	shutdownSignal   chan string
+	addr             string
+	path             string
 }
 
-func (h *callbackEndpoint) start() {
+func (h *callbackEndpoint) setAddr(addr, defaultAddr string) {
```

Comment:
I did not look closely to see if it was viable, but I wonder if the defaults can simply be set by setting http://localhost:9555/callback in the CLI config by default. 

This would simplify things a little bit if it is viable.

Reply:
We do set a default [here](https://github.com/jentz/vigilant-dollop/blob/main/cmd/authorization_code_cfg.go#L25), so I think you are right that we can remove this part and assume we are always provided an address and a path from the CLI config.